packages/init/src/index.ts

```diff
@@ -22,6 +22,7 @@ async function init() {
   const isTrackingAvailable =
     findArgument(process.argv, "tracking") !== "false";
   const maybeRepositorySubdomain = findArgument(process.argv, "repository");
+  const sendDocs = findArgument(process.argv, "no-docs") ? true : false;
```

Comment:
I don't think that would work, did you tested it ?
If I look at the findArgument function it expects a value after that.

Maybe we could make another helper like "findFlag"

Reply:
I'll take a look, and nope I've not had the chance to run because of all the setup required. good catch though :)

Reply:
I've run it now and it deletes the documents, just fine :)